build.gradle

```diff
@@ -283,6 +307,7 @@ allprojects {
         args << '--kotlin_out='
         if (kotlinPackage != null) args[-1] += "kotlin_package=$kotlinPackage,"
         if (logLevel != null) args[-1] += "log=$logLevel,"
+        args[-1] += 'json_use_proto_names=true,'
```

Comment:
📌 For the future refactor of this code: I think the type of casing to use for JSON output should be a runtime option passed to the JSON serializer, not a compile-time option passed to the code generator. Otherwise you'll end up with JSON output that can contain mixed casings if you depend on protobuf messages from a library that was compiled with a different value for this flag.

Reply:
Yes, and we can do that easier with the other serialization strategy we discussed earlier.  Using the `@SerialName` annotation, we actually have to choose what format the well-known types will use when publishing `pbandk`, since the files are committed... this current hard-coded value is indicative of the fact that only streem can/should use `pbandk` as it exists right now.